tests/ui/closures/closure-clone-requires-captured-clone.stderr

```diff
@@ -1,14 +1,14 @@
-error[E0277]: the trait bound `NonClone: Clone` is not satisfied in `{closure@$DIR/closure-clone-requires-captured-clone.rs:13:19: 13:26}`
-  --> $DIR/closure-clone-requires-captured-clone.rs:17:13
+error[E0277]: the trait bound `NonClone: Clone` is not satisfied in `{closure@$DIR/closure-clone-requires-captured-clone.rs:14:19: 14:26}`
+  --> $DIR/closure-clone-requires-captured-clone.rs:18:13
    |
 LL |     let closure = move || {
-   |                   ------- within this `{closure@$DIR/closure-clone-requires-captured-clone.rs:13:19: 13:26}`
+   |                   ------- within this `{closure@$DIR/closure-clone-requires-captured-clone.rs:14:19: 14:26}`
 ...
 LL |     closure.clone();
-   |             ^^^^^ within `{closure@$DIR/closure-clone-requires-captured-clone.rs:13:19: 13:26}`, the trait `Clone` is not implemented for `NonClone`
+   |             ^^^^^ within `{closure@$DIR/closure-clone-requires-captured-clone.rs:14:19: 14:26}`, the trait `Clone` is not implemented for `NonClone`
```

Comment:
Observation (not about this PR): I almost feel like we should just have `{closure}`, the extra path seems... Not particularly easy to read.